packages/affiliates/libs/coingecko.js

```diff
@@ -7,22 +7,24 @@ module.exports = (host = "https://api.coingecko.com") => {
     return result.data;
   }
 
-  // fetch historic prices for a `contract` denominated in `currency` between timestamp `from` and `to`. Note timestamps
+  // Fetch historic prices for a `contract` denominated in `currency` between timestamp `from` and `to`. Note timestamps
   // are assumed to be js timestamps and are converted to unixtimestamps by dividing by 1000.
-  function chart(contract, currency, from, to) {
+  async function getHistoricContractPrices(contract, currency, from, to) {
```

Comment:
This function name was changed to be more descriptive as well as changing the return type. now the function only returns the prices. Previously it would return exchange volume + total market cap which are not useful for any downstream scripts.

Reply:
could probably just add a new function rather than removing the chart function, ie compose them by having getHistoricalPrices call chart, and then checking for .prices and returning that. This is mainly to not break compatibility if some thing is relying on chart to exist. Not a big deal though at this stage.